### PR TITLE
docs: update the extensions guide

### PR DESCRIPTION
**Pull request by OrKoN**

### CODE REVIEW

**Comment by OrKoN** [comment]
@oliverdunk PTAL

**Review by Lightning00Blade — APPROVED** [review]

File: docs/guides/chrome-extensions.md [hunk 1]
```diff
@@ -42,8 +39,40 @@ Chrome Manifest V3 extensions have a background ServiceWorker of type
...
+const pathToExtension = path.join(process.cwd(), 'my-extension');
+const browser = await puppeteer.launch({
+  args: [
+    `--disable-extensions-except=${pathToExtension}`,
+    `--load-extension=${pathToExtension}`,
+  ],
+});
+
+const workerTarget = await browser.waitForTarget(
+  target => target.type() === 'service_worker' && idMatches(target)
```
Comment (oliverdunk) [inline docs/guides/chrome-extensions.md]: I think we should probably remove the `idMatches` call here, unless we want to document how to have an extension with a consistent ID (which seems a little out of scope?).

File: docs/guides/chrome-extensions.md [hunk 2]
```diff
@@ -42,8 +39,40 @@ Chrome Manifest V3 extensions have a background ServiceWorker of type
...
+
+// Test the popup page as you would any other page.
+
+await browser.close();
+```
+
 :::note
 
-It is not yet possible to test extension popups or content scripts.
+It is not yet possible to test extension content scripts.
```
Comment (oliverdunk) [inline docs/guides/chrome-extensions.md:85]: Do you know what the content script callout here was referring to? Or to put it another way, what would we want to exist before we could remove this?

File: docs/guides/chrome-extensions.md [hunk 3]
```diff
@@ -4,8 +4,8 @@ Puppeteer can be used for testing Chrome Extensions.
```
Comment (oliverdunk) [inline docs/guides/chrome-extensions.md:4]: Not sure if we want to link here which has some more advice: https://developer.chrome.com/docs/extensions/how-to/test/end-to-end-testing

**Review by oliverdunk — APPROVED** [review]
Thanks Alex! I left a couple of notes but generally LGTM.

File: docs/guides/chrome-extensions.md (hunk 2 above)
Comment (OrKoN) [reply docs/guides/chrome-extensions.md:85]: I think it is about executing code in the content scripts created by extensions

**Pull request merged**